ConnectionManager.java

```
@@ -0,0 +1,3 @@
+import java.sql.*;
+public class ConnectionManager{
+    public static Connection getConnection(){try{Connection con=DriverManager.getConnection("jdbc:mysql://localhost:3306/test","root","root");return con;}catch(Exception e){}return null;}}
```

The catch block is swallowing exceptions without any handling or logging. This can hide bugs.